### PR TITLE
feat(content): Description improvements part 1

### PR DESCRIPTION
**Pull request by bene-dictator**
This PR adds improvements for ship and outfit descriptions for the Bunrodea, Coalition, Heliarchy, Gegno, Hai, and Unfettered Hai.

### CODE REVIEW

File: data/gegno/gegno vi outfits.txt [hunk 1]
```diff
@@ -10,7 +10,7 @@ outfit "Limescale Breach Sealant"
 	"hull repair rate" .3
 	"hull energy" .15
 	"hull heat" .3
-	description `This is the first generation of the Gegno Vi's "breach sealant" outfits, a relatively new technology that allows a damaged Vi ship to seal any larger wounds in its hull over time. This system is very low-tech, relying on small welding arms placed throughout the ship, but the ability to recover mid-flight is a game-changer in Gegno space. Sadly, most Gegno Vi captains don't seem convinced that the tradeoff of firepower is worth the change.`
+	description `This is the first generation of the Gegno Vi's "breach sealant" outfits, a relatively new technology that allows a damaged Vi ship to seal any larger wounds in its hull over time. This system is very low-tech, relying on small welding arms placed throughout the ship, but the ability to recover mid-flight is a game-changer in Gegno space. Sadly, most Gegno Vi captains don't seem convinced that the tradeoff in firepower is worth the capacity to repair mid-flight.`
```
Comment (AvianGeneticist) [inline data/gegno/gegno vi outfits.txt]: I feel like "capability" is a better word than "capacity".

File: data/hai/unfettered weapons.txt [hunk 2]
```diff
@@ -32,7 +32,7 @@ outfit "Charge Gas Grenadier"
 		"firing energy" 100
 		"firing heat" 80
 		"firing force" 20
-	description `The Unfettered have had no qualms with tweaking Hai technology for their own uses, and this weapon is a good example of that. Essentially an ionic grenade launcher, the simple fabricated canisters fired are under high pressure and likely to explode long before they reach their target, but the unfettered's actions to reduce the size as much as possible by cutting out some of the tanks have resulted in a potent fighter weapon.`
+	description `The Unfettered have made a point of tweaking and improving Hai technology for their own uses, and this weapon is a good example of this strategy. Essentially an ionic grenade launcher, this weapon fires Cargo Gas canisters fired are under high pressure, making them likely to explode long before they reach their target. By re-engineering the Charge Gas Sprayer into a smaller weapon and cutting out an integrated storage tank, the Unfettered have created a potent fighter weapon out of a system typically reserved for larger ships.`
```
Comment (AvianGeneticist) [inline data/hai/unfettered weapons.txt]: "Cargo Gas"? I think you mean "Charge Gas".

**Review by AvianGeneticist — CHANGES_REQUESTED** [review]

File: data/gegno/gegno vi outfits.txt (hunk 1 above)
Comment (bene-dictator) [reply data/gegno/gegno vi outfits.txt]:
```suggestion
	description `This is the first generation of the Gegno Vi's "breach sealant" outfits, a relatively new technology that allows a damaged Vi ship to seal any larger wounds in its hull over time. This system is very low-tech, relying on small welding arms placed throughout the ship, but the ability to recover mid-flight is a game-changer in Gegno space. Sadly, most Gegno Vi captains don't seem convinced that the tradeoff in firepower is worth the capability to repair mid-flight.`
```

File: data/hai/unfettered weapons.txt (hunk 2 above)
Comment (bene-dictator) [reply data/hai/unfettered weapons.txt]:
```suggestion
	description `The Unfettered have made a point of tweaking and improving Hai technology for their own uses, and this weapon is a good example of this strategy. Essentially an ionic grenade launcher, this weapon fires Carge Gas canisters fired are under high pressure, making them likely to explode long before they reach their target. By re-engineering the Charge Gas Sprayer into a smaller weapon and cutting out an integrated storage tank, the Unfettered have created a potent fighter weapon out of a system typically reserved for larger ships.`
```

Comment (AvianGeneticist) [reply data/hai/unfettered weapons.txt]: Well, you're getting closer. It says "Carge Gas" now.

Comment (bene-dictator) [reply data/hai/unfettered weapons.txt]: AAAAAARGH

**Pull request merged**